doc/source/development/extending.rst

```diff
@@ -329,21 +329,11 @@ Each data structure has several *constructor properties* for returning a new
 data structure as the result of an operation. By overriding these properties,
 you can retain subclasses through ``pandas`` data manipulations.
 
-There are 3 constructor properties to be defined:
+There are 3 possible constructor properties to be defined on a subclass:
 
-* ``_constructor``: Used when a manipulation result has the same dimensions as the original.
-* ``_constructor_sliced``: Used when a manipulation result has one lower dimension(s) as the original, such as ``DataFrame`` single columns slicing.
-* ``_constructor_expanddim``: Used when a manipulation result has one higher dimension as the original, such as ``Series.to_frame()``.
-
-Following table shows how ``pandas`` data structures define constructor properties by default.
-
-===========================  ======================= =============
-Property Attributes          ``Series``              ``DataFrame``
-===========================  ======================= =============
-``_constructor``             ``Series``              ``DataFrame``
-``_constructor_sliced``      ``NotImplementedError`` ``Series``
-``_constructor_expanddim``   ``DataFrame``           ``NotImplementedError``
-===========================  ======================= =============
+* ``DataFrame/Series._constructor``: Used when a manipulation result has the same (sub-)class as the original.
```

Comment:
I would keep the "same dimension as the original", because it doesn't necessarily need to return the same class (eg a subclass' `_constructor` can return a non-subclassed DataFrame in certain cases)

Reply:
I didn't like "same dimension as the original" because it could be understood like e.g. an array could be used, which would be wrong. By using "(sub-)class" is is implied that for e.g. for Series you have to supply a Series or a subclass of a Series (but not a DataFrame or a 1-dim array).

Reply:
Is this ok? My opinion is that it’s better to emphazise the type Here, rather than the dimensionality.

Reply:
I’ve changed this to say dimension rather than subclass. IDT the next gwo lines shares this issue, but let me know.